### ISSUE TITLE
{{ "}}" }} breaks the variable parsing

### ISSUE THREAD

**Issue by wolfie** [wolfie/hyde#1]

**Comment by wolfie** [comment]
It's because of the regex parser:

``` javascript
var openTagRegex = /\\{|({{[\s]*(.+?)[\s]*}})|({%[\s]*(.+?)[\s]*%})/g;
```

Need to convert this into a scanner :(

**Comment by wolfie** [comment]
Fixed by 05d2a868e8fa9efed7be49c33ec7cc8e6effb33b
